packages/@aws-cdk/aws-ecs/test/ec2/integ.event-task.lit.ts

```diff
@@ -0,0 +1,48 @@
+import ec2 = require('@aws-cdk/aws-ec2');
+import events = require('@aws-cdk/aws-events');
+import cdk = require('@aws-cdk/cdk');
+import ecs = require('../../lib');
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'aws-ecs-integ-ecs');
+
+const vpc = new ec2.VpcNetwork(stack, 'Vpc', { maxAZs: 1 });
+
+const cluster = new ecs.Cluster(stack, 'EcsCluster', { vpc });
+cluster.addDefaultAutoScalingGroupCapacity({
+  instanceType: new ec2.InstanceType('t2.micro')
+});
+
+/// !show
+// Create a Task Definition for the container to start
+const taskDefinition = new ecs.Ec2TaskDefinition(stack, 'TaskDef');
+taskDefinition.addContainer('TheContainer', {
+  image: ecs.ContainerImage.fromAsset(stack, 'EventImage', { directory: 'eventhandler-image' }),
+  memoryLimitMiB: 256,
+  logging: new ecs.AwsLogDriver(stack, 'TaskLogging', { streamPrefix: 'EventDemo' })
+});
+
+// An EventRule that describes the event trigger (in this case a scheduled run)
+const rule = new events.EventRule(stack, 'Rule', {
+  scheduleExpression: 'rate(1 minute)',
+});
+
+// Use Ec2TaskEventRuleTarget as the target of the EventRule
+const target = new ecs.Ec2TaskEventRuleTarget(stack, 'EventTarget', {
```

Comment:
I am wondering if our CWE pattern should change to always require an "integration" type like this. Seems like the current model is too naive.